src/type-helper.d.ts

```diff
@@ -87,6 +87,14 @@ interface JQuery {
   allTextNodes(exclusions: string[]): Text[];
 }
 
+type BiblioData = { aliasOf?: string; id?: string };
```

Comment:
I prefer `interface` here, IMO `type` is more about creating aliases or complex types.